File: 백준/Bronze/10810. 공 넣기/공 넣기.c
#include <stdio.h>

int main(void) {
    int N, M;
    int arr[101] = { 0 };
    
    scanf("%d%d", &N, &M);
    
    for (int i = 0; i < M; i++) {
        int s, e, num;
        scanf("%d%d%d", &s, &e, &num);
        for (int j = s; j <= e; j++) arr[j] = num;
    }
    
    for (int i = 1; i <= N; i++) {
        printf("%d ", arr[i]);
    }
    
    return 0;
}

File: 백준/Bronze/10810. 공 넣기/공 넣기.py
import sys

n, m = map(int, input().split())
arr = [0 for _ in range(n)]
for _ in range(m):
    a, b, num = map(int, input().split())
    for i in range(a, b + 1):
        arr[i - 1] = num
        
print(*arr)